lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -203,6 +211,21 @@ utxoInternal = toEnum $ fromEnum UTxOInternal
 mutableAccount :: Index 'Soft 'RoleK
 mutableAccount = toEnum $ fromEnum MutableAccount
 
+zeroAccount :: Index 'Soft 'AddressK
+zeroAccount = toEnum 0
```

Comment:
It'd be better to rely on the `Bounded` instance here to avoid hard-coding magic numbers :+1: